docs/concepts/Security-Best-Practices.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Best Practices for a Secure Supply Chain
+description: Best practices for securing your supply chain software using NuGet & GitHub.
+author: JonDouglas
+ms.author: jodou
+ms.date: 02/08/2021
+ms.topic: conceptual
+---
+
+# Best Practices for a Secure Supply Chain
+
+Open Source is everywhere. It is in many proprietary codebases and community projects. For organizations and individuals, the question today is not that you are or are not using open-source code, but what open-source code you are using, and how much.
+
+If you are not aware of what is in your software supply chain, an upstream vulnerability in one of your dependencies can be fatal, making you, and your customers, vulnerable to a potential compromise. In this document, we will dive deeper into what the term “software supply chain” means, why it matters, and how you can help secure your project’s supply chain with best practices.
+
+![The State of the Octoverse 2020 - Open Source](media/opensource-percent.png)
+
+## Dependencies 
+
+The term software supply chain is used to refer to everything that goes into your software and where it comes from. It is the dependencies and properties of your dependencies that your software supply chain depends on. A dependency is what your software needs to run. It can be code, binaries, or other components, and where they come from such as a repository or package manager.
```

Comment:
```suggestion
The term software supply chain is used to refer to everything that goes into your software and where it comes from. It is the dependencies and properties of your dependencies that your software supply chain depends on. A dependency is what your software needs to run. It can be code, binaries, or other components, and where they come from, such as a repository or package manager.
```